pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -294,6 +295,22 @@ sleep 5s && ifconfig usb0 down
 
 After this you should be able to easily use your Intel Edison in USB host mode.
 
+## Jetson Devices
+### Custom device trees
+Loading of custom device-trees in balenaOS is supported by the Jetson Nano and Jetson TX2 family of devices. The list of devices that suport this function includes:
+- Jetson Nano SD-CARD
+- Jetson Nano eMMC
+- Jetson Nano 2GB Devkit SD
+- Jetson TX2
+- Jetson TX2 NX (with Xavier NX Devkit)
```

Comment:
We should avoid having to maintain this list twice. Maybe remove it from the troubleshooting guide and just link to here for all the information?

Reply:
Kyle and I provided the same review @acostach If you ask me, I would rather prefer to not have the troubleshooting guide. So you can follow the suggestion I provided below, and keep only instance of this information.

Reply:
sure, will do